client/src/app/login/login.component.css

```
@@ -0,0 +1,85 @@
+* {
+    font-family: 'JetBrains Mono', monospace;
+}
+
+.login-form-container {
+        flex-direction: column;
+        justify-content: center;
+        text-align: center;
+        width: 434px;
+        margin: 15% auto;
```

I would leave 
```suggestion
        margin: auto;
```
take the 15% away, because now when you resize screen, the vertical position changes.